packages/fxa-shared/nestjs/sentry/reporting.ts

```diff
@@ -122,6 +122,10 @@ export function reportRequestException(
   excContexts: ExtraContext[] = [],
   request?: Request
 ) {
+  // Don't report already reported exceptions
+  if ((exception as any).reported) {
```

Comment:
Maybe `exception`'s type could be changed to `exception: Error & { reported: boolean }` so we don't need to cast to `any`?

Reply:
Oh, nice call! Updated the type to avoid the casts.